master/getting-started/kubernetes/installation/integration.md

```diff
@@ -79,17 +79,17 @@ ExecStart=/usr/bin/docker run --net=host --privileged --name={{site.noderunning}
   -e ETCD_ENDPOINTS=${ETCD_ENDPOINTS} \
   -e NODENAME=${HOSTNAME} \
   -e IP= \
-  -e NO_DEFAULT_POOLS= \
-  -e AS= \
-  -e CALICO_LIBNETWORK_ENABLED=true \
   -e IP6= \
+  -e AS= \
+  -e NO_DEFAULT_POOLS= \
+  -e CALICO_LIBNETWORK_ENABLED=false \
```

Comment:
Is removing CALICO_LIBNETWORK_ENABLED the same as setting it false?  (I see it isn't present in the v3.1)
If it is the same then probably just dropping it would be my vote, unless it is staying as a placeholder for the future.

Reply:
yeah, I'm leaving it as a placeholder for now. It should be set to false anyway when using k8s.